image_width: 1920
image_height: 1200
CameraExtrinsicMat:
   rows: 4
   cols: 4
   dt: d
   data: [ 3.1355815316385094e-02, 2.1617143720422449e-02,
       9.9927449279125247e-01, 2.6289999720461460e-01,
       -9.9315683435152091e-01, -1.1185546026650606e-01,
       3.3583602984687821e-02, 3.7146587960692527e-02,
       1.1250028989611477e-01, -9.9348933316162880e-01,
       1.7961894870851092e-02, -3.0568771956851554e-02, 0., 0., 0., 1. ]
CameraMat:
   rows: 3
   cols: 3
   dt: d
   data: [ 1.0345529146135275e+03, 0., 9.8136181442398549e+02, 0.,
       1.0337238631425546e+03, 6.1348644049507277e+02, 0., 0., 1. ]
DistCoeff:
   rows: 1
   cols: 5
   dt: d
   data: [ -1.1796704919938258e-01, 6.4477549186636340e-02,
       3.3270251117984849e-04, 1.2305246304840027e-03,
       -1.3514633513756615e-02 ]
